src/app/containers/VotePage/index.tsx

```diff
@@ -1,15 +1,85 @@
 import React from 'react';
 
 import { Helmet } from 'react-helmet-async';
+import { Link } from 'react-router-dom';
+import { Box } from 'rebass/styled-components';
+import styled from 'styled-components';
 
+import { Button } from 'app/components/Button';
 import { DefaultLayout } from 'app/components/Layout';
+import ProposalInfo from 'app/components/ProposalInfo';
+import { Typography } from 'app/theme';
+import { comparatorDesc } from 'utils';
+
+const metadata = {
+  voted: 16,
+  voters: 748,
+  approvePercentage: 53,
+  rejectPercentage: 14,
+  timestamp: 284400000,
+};
 
 export function VotePage() {
+  const VoteContainer = styled(Box)`
+    flex: 1;
+    border-radius: 10px;
+    padding: 35px 35px;
+    background-color: ${({ theme }) => theme.colors.bg2};
+    margin-bottom: 50px;
+  `;
+
+  const VoteHeader = styled(Box)`
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(128px, 1fr));
+    margin-bottom: 30px;
+  `;
+
```

Comment:
the `mockData` needs to be out of scope. the variable defines and allocates memory for every component render. it is not good.